_content/articles/yunnan-burma-frontier-since-the-end-of-17th-century_jianxiong-ma.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Shaping of the Yunnan-Burma Frontier by Secret Societies since the End of the 17th Century"
+authors:
+  - "Ma Jianxiong"
+external_url: "https://journals.openedition.org/moussons/538"
+drive_links:
+  - "https://drive.google.com/file/d/15J73T2aXJHyxoglIUf5zi5TshbLaECtt/view?usp=sharing"
+course: burmese
+tags:
+  - state
```

Comment:
`sea` for sure.  🤔 Let's also add `qing` and `yunnan`...  I know they aren't tags, but they are now! 😊